src/h_vialib/secure/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Security helpers."""
+
+from h_vialib.secure.expiry import quantized_expiry
+from h_vialib.secure.token import SecureToken, SecureURLToken, ViaSecureURLToken
```

Comment:
Can we add this stuff to `h-vialib/__init__.py`, like we do with `Configuration`: <https://github.com/hypothesis/h-vialib/blob/master/src/h_vialib/__init__.py>?

Reason: flat is better than nested. I just want to import what I need from `h_vialib`. I don't want to have to remember what comes from `h_vialib.configuration`, what comes from `h_vialib.secure`, etc. Instead of:

```python
from h_vialib.configuration import Configuration
from h_vialib.secure import ViaSecureURLToken
```

It could be:

```python
from h_vialib import Configuration, ViaSecureURLToken
```

Reply:
Maybe.  There will pretty soon be about 8 things in `__init__.py`, we might not want to put everything there for the same reason we don't keep all files in the root of our projects. Flat is better than nested except when it's the same or worse.

I'd also suggest that there are two distinct customers for `h-vialib`'s public interface:

* Actual types of Via product
* Client types (LMS, h, client?)

As a client, you maybe shouldn't be poking about into nested directories, despite them being "public". This gives us a way to hint at that. This also means we trade one type of confusion "where do I import things from?" for another "how many of these classes am I supposed to use as a client?" The difference being that Via type users are high information users, and can be expected to know. 

It's worth keeping both of these "problems" in proportion though. You need to "remember" which to use or where to get it from for the short period of time, and as long as it's not hard to find out, then git will do the remembering for you afterwards (Do you remember exactly where `HTTPFound` comes from in `pyramid`? Maybe maybe not, but it's not hard to find out). I'd also recommend looking into hooking up some autocomplete in your IDE. I press "Alt + Enter" on a class name and Pycharm does a reasonable job of importing it for me _most_ of the time.

As we ship in and evolve the interface, we might want move the other way and hide `Configuration` from the root, as clients shouldn't need it, so there might only be two or three classes.